sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -247,10 +245,10 @@
      * @param form The data of the form to recognize content information from.
      * @param length The exact length of the data. Size of the file must be less than 50 MB.
      * @param formContentType The type of the provided form. Supported Media types including .pdf, .jpg, .png or
-     * .tiff type file stream.
+     * .tiff type file stream. Content-type is auto-detected and could be {@code null}.
      *
-     * @return A {@link PollerFlux} that polls the recognize receipt operation until it has completed, has failed,
-     * or has been cancelled. The completed operation returns a List of {@link FormPage}.
+     * @return A {@link PollerFlux} polls the recognize layout operation until it has completed, has failed, or has
```

Comment:
any reason we use layout instead of `recognized content operation`? similar to what Krista said above